src/Microsoft.IdentityModel.Clients.ActiveDirectory/AuthenticationContext.cs

```diff
@@ -321,6 +317,8 @@ public Task<AuthenticationResult> AcquireTokenByDeviceCodeAsync(DeviceCodeResult
         /// <param name="claims">Additional claims that are needed for authentication. Acquired from the AdalClaimChallengeException</param>
         /// <param name="synchronizationContext">Optional SynchronizationContext for ensuring interactive calls are executed on the UI thread.</param>
         /// <returns>It contains Access Token and the Access Token's expiration time.</returns>
+        [Obsolete("Microsoft.IdentityModel.Clients.ActiveDirectory has been deprecated and replaced with Microsoft.Identity.Client. " +
```

Comment:
I am not sure if this one can be used from PCA and CCA, so I added a general purpose migration url